### PR TITLE
Editing the version in footer using new VersionFactory

### PR DESCRIPTION
**Pull request by sonejah21**

### CODE REVIEW

File: src/client/app/components/footer/footer.directive.js [hunk 1]
```diff
@@ -8,9 +8,37 @@
...
+      function activate() {
+        updateVersion();
+      }
+
+      function updateVersion() {
+        Version.get().$promise.then(handleResults);
+
+        function handleResults(version) {
+          vm.version = version.jellyfish_version;
+          console.log(version.jellyfish_version);
```
Comment (stackus) [inline src/client/app/components/footer/footer.directive.js]: logging

File: src/client/app/components/footer/footer.directive.js [hunk 2]
```diff
@@ -8,9 +8,37 @@
...
+      link: link,
+      templateUrl: 'app/components/footer/footer.html',
+      controller: FooterController,
+      controllerAs: 'vm',
+      bindToController: true
     };
 
     return directive;
+
+    function link(scope, element, attrs, vm, transclude, Version) {
```
Comment (stackus) [inline src/client/app/components/footer/footer.directive.js]: Version cannot be made available here. This function isn't injected.

File: src/client/app/components/footer/footer.directive.js [hunk 3]
```diff
@@ -8,9 +8,37 @@
...
+      vm.activate();
+    }
+
+    /** @ngInject */
+    function FooterController($q, Version) {
```
Comment (stackus) [inline src/client/app/components/footer/footer.directive.js]: Remove `$q` now that is is not being used. The IDE should be pointing these out in the form of warning inspections (slim lines on the very right edge of the window)

Comment (sonejah21) [inline src/client/app/components/footer/footer.directive.js]: Good tip! I'll be watching for these. I never noticed them before.

**Pull request merged**